src/python/CRABClient/Commands/status.py

```diff
@@ -57,7 +60,7 @@ def __call__(self):
         if not rootDagId:
             failureMsg = "The task has not been submitted to the Grid scheduler yet. Not printing job information."
```

Comment:
please see https://github.com/dmwm/CRABClient/issues/4702#issuecomment-304273801
We should not depend on explanations in comments here in order to argue on the meaning of things.